deploy/olm-catalog/portworx/99.9.9/portworx-certified.clusterserviceversion.yaml

```diff
@@ -10,7 +10,7 @@ metadata:
     containerImage: portworx/px-operator:master
     repository: https://github.com/libopenstorage/operator
     createdAt: 2022-01-01T08:00:00Z
-    support: Portworx, Inc
+    support: https://docs.portworx.com/portworx-enterprise/support/contact-support
```

Comment:
I see why this link is not working because this field is not supposed to be a link. Let's just put `Pure Storage, Inc` here.

The support link is displayed from the links section below.

Reply:
Updated